### PR TITLE
Update Acme and Nord Light for git gutters

### PR DESCRIPTION
**Pull request by two-six**
nord light diffs
acme diff
Fixes for #4972

### CODE REVIEW

File: runtime/themes/nord_light.toml [hunk 1]
```diff
@@ -58,6 +58,10 @@
 "markup.link.text" = {fg="nord12"}
 "markup.quote" = {fg="nord3", modifiers=["italic"]}
 
+"diff.plus" = {fg = "nord14"}
+"diff.delta" = {fg = "nord11"}
+"diff.minus" = {fg = "nord15"}
```
Comment (archseer) [inline runtime/themes/nord_light.toml:63]:
```suggestion
"diff.plus" = {fg = "nord14"}
"diff.delta" = {fg = "nord13"}
"diff.minus" = {fg = "nord11"}
```

This is per descriptions in https://www.nordtheme.com/docs/colors-and-palettes

**Comment by pascalkuthe** [comment]
It's a bit odd that modified is red and removed is blue (for the second theme). That's pretty much the opposite of most other themes/the general expectation. Would it be an option to swap those two? (same for the other theme although I think it's still ok this way)

File: runtime/themes/acme.toml [hunk 2]
```diff
@@ -22,6 +22,9 @@
 "diagnostic.hint" = {bg="white", modifiers=["bold"]}
 "ui.bufferline" = { fg = "indent", bg = "acme_bar_bg" }
 "ui.bufferline.active" = { fg = "black", bg = "acme_bg" }
+"diff.plus" = {fg = "green"}
+"diff.delta" = {fg = "red"}
+"diff.minus" = {fg = "acme_bar_bg"}
```
Comment (archseer) [inline runtime/themes/acme.toml:27]: Same here, you probably want minus as red (deletion) and delta as something else (modification, usually purple? `acme_bar_bg` could work)

**Review by archseer — CHANGES_REQUESTED** [review]

**Comment by two-six** [comment]
> It's a bit odd that modified is red and removed is blue (for the second theme). That's pretty much the opposite of most other themes/the general expectation. Would it be an option to swap those two? (same for the other theme although I think it's still ok this way)

Sorry, I was taking care of it late at night so I didn't notice. You're completely right 

@archseer Thanks for this change, it looks better :)

**Pull request merged**